File: leetcode1.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
     
    for (let i =0;i<nums.length;i++){
        let diff=target-nums[i];
        for(let j=i+1;j<nums.length;j++){
            if (nums[j]==diff){
                
                if (i!=j){
                    return [i,j];
                }
              }
         
        }
    }//something
    
    
};

File: solution_in_python/leetcode1.py
#this is solution of 2sum
class Solution(object):
    def twoSum(self, nums, target):
        idx=0
        for i in range(len(nums)):
            diff=target-nums[i]
            if diff in nums :
                idx=nums.index(diff)
                if i!=idx:
                    return i,idx
                    


                